content/reference/cli/git-comparison.md

```diff
@@ -81,13 +81,14 @@ We aim to match our CLI command behavior as closely to their Git equivalent as p
 
 ## Administration
 
-| Component       | Supported | Notes and limitations                                                                 |
-| :-------------- | :-------- | :------------------------------------------------------------------------------------ |
-| `clean`         | 🟠        | [does not completely match git behavior](https://github.com/dolthub/dolt/issues/6313) |
-| `gc`            | ✅        |                                                                                       |
-| `fsck`          | ❌        |                                                                                       |
-| `reflog`        | ❌        |                                                                                       |
-| `filter-branch` | ✅        |                                                                                       |
+| Component       | Supported | Notes and limitations                                                                                                                                                                             |
+|:----------------|:----------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `clean`         | 🟠        | [does not completely match git behavior](https://github.com/dolthub/dolt/issues/6313)                                                                                                             |
+| `gc`            | ✅         |                                                                                                                                                                                                   |
+| `fsck`          | ❌         |                                                                                                                                                                                                   |
+| `reflog`        | 🟠         | [does not completely match git behavior](https://www.dolthub.com/blog/2023-11-17-dolt-reflog/):<br/> <li>only supports named references (not Git's special refs)</li> <li>shows deleted refs</li> |
```

Comment:
I'm slightly torn between ✅ and 🟠 on this one. It's true that Git's reflog has a few more features than we have, but I'm not sure they're super important for customers. I might slightly lean towards ✅ on this one and then adjusting if we get feature requests for gaps in reflog support. I really like how for `git clean` we have a concrete GH issue that we link to with the gap. 

I know you've been maintaining this list, so if you feel strongly that this fits into an 🟠 better, based on the other ratings we have, then I'm on board with that.